### PR TITLE
Celeritas: new version 0.4.3

### PR DESCRIPTION
**Pull request by sethrj**

### CODE REVIEW

**Review by tldahlgren — APPROVED** [review]
Confirmed the version sha256.

**Comment by tldahlgren** [comment]
Will wait for CI *and* @drbenmorgan 's feedback before merging.

**Comment by sethrj** [comment]
I fucked up this release and tagged the wrong branch. Please disregard.

**Pull request closed without merging**
